sidebars.js

```diff
@@ -178,6 +207,5 @@ module.exports = {
         "opensauced-packages/semantic-release",
       ],
     },
-    
```

Comment:
community guidlines and procedures

Reply:
@BekahHW, what do you think of this suggestion?